zebra-chain/src/serialization/display_order.rs

```diff
@@ -0,0 +1,34 @@
+//! Trait for converting types to their byte representation for display or RPC use.
+///
+/// This trait provides methods to access and construct types from their
+/// internal serialized byte order (typically little-endian) as well as their
+/// big-endian byte order used for display and RPC.
+pub trait BytesInDisplayOrder<
+    const SHOULD_REVERSE_BYTES_IN_DISPLAY_ORDER: bool = false,
+    const BYTE_LEN: usize = 32,
```

Comment:
It would be nice to use this for `ExpandedDifficulty`, `CompactDifficulty`, and `ConsensusBranchId` too now that it can return varying array sizes.